src/components/PasswordField/PasswordField.tsx

```diff
@@ -28,7 +28,7 @@ import {
 } from '../../channels/SavedPasswordsChannels/SavedPasswordsChannels';
 import { RouteExcept } from '../RouteExcept/RouteExcept';
 import { generatePath, paths } from '../../views/paths';
-import { exceptionToError } from '../../utilities/exceptionToError/exceptionToError';
+import { PasswordError } from '../../utilities/storageEncryption/storageEncryption';
```

Comment:
I suggest to re-export it from here as well, so that we have fewer imports in the other files